### DISCUSSION TITLE
Keyboard Input

### DISCUSSION THREAD

**Discussion by aaronculp-droneup** [soywod/react-pin-field#81] [General]
Curious how to wire this up with buttons instead of inputs....

I would like to use this package for 1-9 buttons a user can press to create the pin

**Comment by soywod** [comment]
> I would like to use this package for 1-9 buttons a user can press to
> create the pin

I am afraid this is out of scope of react-pin-field. The aim of the lib
is to type a code, using a keyboard. What you need is buttons to press
or touch (kind of simulating a keyboard). This can be easily done on its
own, just need 9 buttons with onClick handlers that update the same text
input.
